demos/human_pose_estimation_demo/python/human_pose_estimation_demo.py

```diff
@@ -85,6 +85,9 @@ def build_argparser():
 
     io_args = parser.add_argument_group('Input/output options')
     io_args.add_argument('-no_show', '--no_show', help="Optional. Don't show output.", action='store_true')
+    io_args.add_argument('--output_resolution', default=None, type=str,
```

Comment:
I recommend to use custom type, which would parse results and handle errors, e.g.:
```python
io_args.add_argument('--output_resolution', default=None, type=resolution,
```
```python
def resolution(value):
    try:
        result = [int(v) for v in args.output_resolution.split('x')])
        if len(result) != 2:
            raise ArgumentError('...')
    except ...:
        ...
    return result
```

Reply:
So, you need check to_int conversion and right shape.

Reply:
Done